### PR TITLE
Discovery Module and Poller Module configuration via Global Settings Web GUI

### PR DESCRIPTION
**Pull request by SourceDoctor**
configure global Settings from Discovery Module and Poller Module via GUI

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: html/mix-manifest.json [hunk 1]
```diff
@@ -1,12 +1,8 @@
 {
-    "/js/app.js": "/js/app.js?id=d074dd82ac08dba78c44",
```
Comment (Jellyfrog) [inline html/mix-manifest.json]: This seems wrong

Comment (SourceDoctor) [reply html/mix-manifest.json]: @Jellyfrog 
i ran this command `lnms translation:generate` and this diff from mix-manifest.json is the result
Maybe i miss something?

Comment (Jellyfrog) [reply html/mix-manifest.json]: works for me, did you do "npm install" before?

Comment (SourceDoctor) [reply html/mix-manifest.json]: ok i did `npm install` and runs lnms translation again now in diff only

vendor.js and <language>.js is changed

should it be like this? If so, i'll commit it :-)

Comment (SourceDoctor) [reply html/mix-manifest.json]: @Jellyfrog fixed it .. hope so :-)

**Comment by SourceDoctor** [comment]
@Jellyfrog something else to fix here?

File: resources/lang/en/settings.php [hunk 2]
```diff
@@ -324,6 +326,135 @@
             'description' => 'Device performance entries older than (days)',
             'help' => 'Cleanup done by daily.sh'
         ],
+        'discovery_modules' => [
+            'arp-table' => [
+                'description' => 'arp-table'
```
Comment (Jellyfrog) [inline resources/lang/en/settings.php]: All these descriptions could be a bit better, `ARP Table` for example

Comment (SourceDoctor) [reply resources/lang/en/settings.php]: done

**Comment by Jellyfrog** [comment]
Wondering if "Poller Modules" should go under "Poller" instead of "Discovery"  
Thoughts @librenms/reviewers ?

**Comment by SourceDoctor** [comment]
@Jellyfrog 
good point
i moved "Poller Modules" to "Poller" Category

**Comment by SourceDoctor** [comment]
@Jellyfrog approved but not merged? ;-)

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-60-release-changelog-january-2020/10863/1

**Pull request merged**